src/templates/registration/login.html

```diff
@@ -2,66 +2,47 @@
 {% load static %}
 
 {% block content %}
-    <div class="ui middle aligned center aligned grid registration_top_padding">
+    <div class="ui middle aligned center aligned grid">
         <div class="column">
-            <h2 class="ui teal image header">
-                {#                <img src="assets/images/logo.png" class="image">#}
-                <div class="content">
-                    Log-in to your account
-                </div>
-            </h2>
             <form class="ui large form" method="POST">
+                <div class="ui divider"></div>
+                <h2 class="ui blue header">
+                    Login
+                </h2>
                 {% csrf_token %}
-
-                <div class="field">
-                    <div class="ui left icon input">
-                        <i class="user icon"></i>
-                        <input type="text" name="username" autofocus="" maxlength="254" required="" id="id_username">
+                <div class="ui stacked segment">
+                    <div class="ui field">
+                        <div class="ui left icon input">
+                            <i class="user icon"></i>
+                            <input type="text"
+                                   name="{{ form.username.name }}"
+                                   placeholder="username"
+                                   id="{{ form.username.id_for_label }}"
```

Comment:
we don't even need this attribute I dont think, pull out as much cruft as you can from here